.github/workflows/test_common.yml

```diff
@@ -1,4 +1,4 @@
-name: test common
+name: common | common
```

Comment:
maybe just name it common?

Reply:
I'm trying to keep the same format across all files